homeassistant/util/yaml.py

```diff
@@ -156,6 +157,21 @@ def _ordered_dict(loader: SafeLineLoader,
     return processed
 
 
+def _construct_seq(loader, node):
```

Comment:
It does something :)
I'm investigating the difference now, but in order to get line numbers on all validation errors I need both at the moment. I will try to get line numbers on all configuration rows in one place.